src/framework/audio/thirdparty/opusenc/cmake/SetupOpus.cmake

```diff
@@ -37,4 +37,8 @@ if (NOT OPUS_FOUND)
 
     set(OPUS_INCLUDE_DIRS ${OPUS_LIB_DIR}/include)
     set(OPUS_LIBRARIES opus)
+
+    target_no_warning(${MODULE} -Wno-conversion)
```

Comment:
Here too, `${MODULE}` is defined to the nearest ancestor folder that uses the module system, which is opusenc. The target you want to specify here is just `opus`.

Reply:
Will try, tomorrow...

Reply:
Thanks, works